### PR TITLE
Improve UI components

### PR DESCRIPTION
**Pull request by anbraten**

### CODE REVIEW

File: packages/app/src/components/uiBlocks/Textfield.vue [hunk 1]
```diff
@@ -35,11 +32,8 @@ export default defineComponent({
...
-    label: {
-      type: String,
-      default: '',
-    },
   },
+
   emits: ['update:modelValue'],
 });
-</script>
+</script>
```
Comment (julianhahn) [inline packages/app/src/components/uiBlocks/Textfield.vue:39]:
```suggestion
</script>

```
Just add a new line to fix the failed lint check

File: packages/app/src/views/Sandbox.vue [hunk 2]
```diff
@@ -13,15 +13,15 @@
...
-      />
+      <LabeledInput label="NFC-Tag">
+        <Dropdown
+          v-model="dropdownValue"
+          class="mb-5 h-20"
+          enable-add-item
+          :items="dropdownItems"
+          @add-item="addDropdownItem"
+          @remove-item="removeDropdownItem"
+      /></LabeledInput>
```
Comment (julianhahn) [inline packages/app/src/views/Sandbox.vue:24]: this looks very weird in the sandbox. What's the intention here?
image

**Review by julianhahn — CHANGES_REQUESTED** [review]

File: packages/app/src/views/Sandbox.vue (hunk 2 above)
Comment (anbraten) [reply packages/app/src/views/Sandbox.vue:24]: I haven't checked that. Seems like the `classes` have to go on the `LabeledInput`

Comment (anbraten) [reply packages/app/src/views/Sandbox.vue]: fixed by 3bc4aa4784c9286bb9085de8d47b9887d745e59b

**Review by JanEberlein — APPROVED** [review]

**Review by tcoro — APPROVED** [review]
looks good to me :)

**Pull request merged**